README.md

```diff
@@ -260,7 +265,7 @@ VALUES (
 select * from test_tidb_type;
 ```
 
-
+If there are primary keys in tidb table, you can enable upsert mode by `SET SESSION tidb.upsert_mode_enable=true`. TiDB primary keys will be mapping as presto table properties `primary_keys`.
```

Comment:
there are primary keys -> there is primary key 
mapping -> mapped
properties -> property
primary_keys -> primary_key